features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/model/TimelineItem.kt

```diff
@@ -52,6 +52,7 @@ sealed interface TimelineItem {
     data class Event(
         val id: String,
         val eventId: EventId? = null,
+        val transactionId: String? = null,
```

Comment:
Does it worth creating a `TransactionId` value class, as per `EventId`?

Reply:
I'm not sure if there is some format we should enforce here. If there isn't, I don't think it's worth it.

Reply:
It just to have more typed code, and so enforce usage at compilation time. Using `String` every where can lead to errors.

Reply:
I don't think this is an important value like `EventId` or `RoomId`, so I'd keep it as `String` IMO.